docs/sanitizing-data.asciidoc

```diff
@@ -1,12 +1,12 @@
 [[sanitizing-data]]
-== Sanitizing data
+=== Sanitizing data
 
-Sometimes it is necessary to sanitize the data sent to Elastic APM, e.g. remove
-sensitive data.
+Sometimes it is necessary to sanitize the data sent to Elastic APM,
+e.g. remove sensitive data.
 
-To do this with the Elastic APM module, you create a processor. A processor is a function
-that takes a `client` instance as well as an event (an error or a transaction), and returns
-the modified event.
+To do this with the Elastic APM module, you create a processor.
+A processor is a function that takes a `client` instance as well as an event (an error or a transaction),
```

Comment:
lingo in this side of the pipeline is still `exception`, not `error` (`capture_exception` etc)